test/api_compatibility.js

```diff
@@ -55,45 +70,23 @@ module.exports = function apiCompatibilityChecks(_createTracer) {
                 });
 
                 it('should handle Spans and SpanContexts', function() {
-                    var tracer = createTracer();
-                    var span = tracer.startSpan('test_operation');
                     var textCarrier = {};
                     expect(function() { tracer.inject(span, opentracing.FORMAT_TEXT_MAP, textCarrier); }).to.not.throw(Error);
                 });
             });
         });
 
         describe('Span', function() {
-            var tracer = createTracer();
-            var span = tracer.startSpan('test_span');
-
-            it('should be a class', function() {
-                expect(span).to.be.an('object');
-            });
-
-            var funcs = [
-                'tracer',
-                'context',
-                'setTag',
-                'addTags',
-                'log',
-                'logEvent',
-                'finish',
-                'setBaggageItem',
-                'getBaggageItem',
-            ];
-            _.each(funcs, function(name) {
+            _.each(spanFunctions, function(name) {
                 it(name + ' should be a method', function() {
                     expect(span[name]).to.be.a('function');
                 });
             });
 
             describe('Span#finish', function() {
                 it('should not throw exceptions on valid arguments', function() {
-                    var tracer = createTracer();
                     function f(arg) {
                         return function() {
-                            var span = tracer.startSpan('test_span');
```

Comment:
don't you want a new span every time? otherwise you're calling finish twice on the same span